### PR TITLE
added hint to cses-1620

### PR DESCRIPTION
**Pull request by RedBlueBird**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
what happened to the other pr lol

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
wait nvm for some reason i can't edit the files?

**Comment by SansPapyrus683** [comment]
what the- i'm trying to edit the file but the changes aren't processing

**Review by SansPapyrus683 — APPROVED** [review]
@ryanchou-dev review

File: solutions/silver/cses-1620.mdx [hunk 1]
```diff
@@ -26,8 +42,11 @@ Overall, the $n$ machines can create
 ‎‎$\sum_{i=1}^{n} \lfloor \frac{ans}{k_i} \rfloor$ products. If this sum
 $\geq t$, then $ans$ is valid.
 
-<LanguageSection>
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(n \log t)$, where $t$ is the largest time we need to consider.
```
Comment (ryanchou-dev) [inline solutions/silver/cses-1620.mdx:47]: $\max(t_i)$ instead?

**Review by ryanchou-dev — APPROVED** [review]
chillin

File: solutions/silver/cses-1620.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/silver/cses-1620.mdx:47]: imo it's fine?

**Review by nwatx — APPROVED** [review]

**Pull request merged**